### PR TITLE
Updates for sunpy sphinx theme v2

### PR DESCRIPTION
**Pull request by Cadair**
This repo had a lot of changes.

### CODE REVIEW

File: about/project.rst [hunk 1]
```diff
@@ -222,8 +222,9 @@ If you would like to get involved please get in touch.
 .. _David Pérez-Suárez: https://github.com/dpshelio
 .. _Kolja Glogowski: https://github.com/kbg
 .. _Jack Ireland: https://github.com/wafels
-.. _Stuart Mumford: https://github.com/cadair
+.. _Stuart Mumford: https://github.com/Cadair
 .. _Albert Shih: https://github.com/ayshih
 .. _Will Barnes: https://github.com/wtbarnes
 .. _Bin Chen: https://github.com/binchensun
 .. _Shane Maloney: https://github.com/samaloney
+.. _Nabil Freij: https://github.com/nabobalis
```
Comment (nabobalis) [inline about/project.rst:230]:
```suggestion
```

File: about/index.rst [hunk 2]
```diff
@@ -1,8 +1,42 @@
+About
+=====
+
 .. toctree::
    :hidden:
 
-   mission
+   self
```
Comment (nabobalis) [inline about/index.rst:7]: this works?!

**Comment by nabobalis** [comment]
https://deploy-preview-386--sunpy.netlify.app/about/mission is in the navbar?

File: about/index.rst (hunk 2 above)
Comment (Cadair) [reply about/index.rst:7]: yes

File: about/project.rst (hunk 1 above)
Comment (Cadair) [reply about/project.rst:230]: You do those roles you really should be listed here...

File: requirements.txt [hunk 3]
```diff
@@ -1,11 +1,12 @@
 ablog>=0.11.0rc1
-sunpy-sphinx-theme
+sunpy-sphinx-theme>=2.0.0rc1
```
Comment (wtbarnes) [inline requirements.txt:2]: Bump this?

**Review by wtbarnes — APPROVED** [review]
This looks good to me, though I mainly reviewed the rendered website and not the code.

**Comment by wtbarnes** [comment]
One annoying thing I found. For some reason in the blog posts written in notebooks, the equations all get right-aligned, e.g. 

image

I suspect this is some weird interaction between the pydata-based theme and the notebook extension.

File: requirements.txt (hunk 3 above)
Comment (Cadair) [reply requirements.txt:2]: This is the min version this build works with :shrug:

**Comment by Cadair** [comment]
> One annoying thing I found. For some reason in the blog posts written in notebooks, the equations all get right-aligned

Fixed in https://github.com/sunpy/sunpy-sphinx-theme/pull/220

https://output.circle-artifacts.com/output/job/3e97e7a4-a30d-4418-ac8c-0d62ab448d8a/artifacts/0/.tmp/website/sunpy.org/_build/html/posts/2018/2018-07-21-coronal-loop-coordinates.html

**Pull request merged**